ui/src/app/pages/data/data-set/export-dataset-modal.tsx

```diff
@@ -101,7 +105,7 @@ export const ExportDatasetModal: (props: Props) => JSX.Element = fp.flow(
       kernelType,
       genomicsAnalysisTool,
       generateGenomicsAnalysisCode: hasWgs(),
-      notebookName,
+      notebookName: appendNotebookFileSuffix(notebookNameWithoutSuffix),
```

Comment:
fixes a bug introduced in https://github.com/all-of-us/workbench/pull/7166/files#r1029816783 (which we are also planning to hotfix by changing that line in the API.  It does not hurt to fix it in both places.